src/components/menu/menu/menu.component.ts

```diff
@@ -167,6 +168,7 @@ export class MenuComponent implements AfterContentInit, OnDestroy, OnChanges {
         this._activeItem$.complete();
         this._items$.complete();
         this._placement$.complete();
+        this._keyManager.tabOut.complete();
```

Comment:
I don't think this is appropriate, you didn't create this observable so you should not terminate it.